.travis/run.sh

```diff
@@ -6,13 +6,15 @@ if [[ "${TOXENV}" == "pypy" ]]; then
     eval "$(pyenv init -)"
 fi
 if [ -n "${LIBRESSL}" ]; then
-    OPENSSL=$LIBRESSL
-    export CFLAGS="-Werror -Wno-error=deprecated-declarations -Wno-error=discarded-qualifiers -Wno-error=unused-function"
+    LIBRESSL_DIR="ossl-2/${LIBRESSL}"
```

Comment:
Why did this branch need to change?

Reply:
Because we were using the OpenSSL code path previously to do the directory path creation. Now that we build the directory with a hash based on the config args that no longer works for libre (where we don't have those config args).